static/js/message_viewport.js

```diff
@@ -307,7 +307,7 @@ exports.is_narrow = function () {
     // This basically returns true when we hide the right sidebar for
     // the left_side_userlist skinny mode.  It would be nice to have a less brittle
     // test for this.
-    return window.innerWidth < media_breakpoints.xl_min;
+    return window.innerWidth < Number.parseInt(media_breakpoints.xl_min.slice(0,-2));
```

Comment:
`media_breakpoints.xl_min` is actually a string `1200px`, that's why this function wasn't giving right output.